doc/users/next_whats_new/view_current_axis_format.rst

```diff
@@ -0,0 +1,16 @@
+View current appearance settings for ticks, tick labels, and gridlines
+----------------------------------------------------------------------
+
+The new `~matplotlib.axis.Axis.get_tick_params` method can be used to
+retrieve the appearance settings that will be applied to any
+additional ticks, tick labels, and gridlines added to the plot:
+
+.. code-block:: python
+
+    import matplotlib.pyplot as plt
+
+    fig, ax = plt.subplots()
+    ax.yaxis.set_tick_params(labelsize=30, labelcolor='red',
+                             direction='out', which='major')
+    print(ax.yaxis.get_tick_params(which='major'))
+    print(ax.yaxis.get_tick_params(which='minor'))
```

Comment:
This code block doesn't really tell me too much by itself.  Can it get the result of the print statements added as well?